### PR TITLE
Insights release of total enrollment

### PR DESCRIPTION
**Pull request by lamagnifica**
@shnayder @dsjen release notes!
@srpearce @catong @mhoeber this feature should be releasing today...

### CODE REVIEW

**Comment by dsjen** [comment]
LGTM! :dancers:

File: en_us/release_notes/source/2015/insights/insights_0806_2015.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+A new metric that measures total interest in your course over time is now
+available in edX Insights. The Enrollment Activity page now includes a metric
```
Comment (srpearce) [inline en_us/release_notes/source/2015/insights/insights_0806_2015.rst]: Bold "Enrollment Activity"

**Comment by srpearce** [comment]
One nit - otherwise :+1:

**Comment by mhoeber** [comment]
Seems potentially confusing that one could see change listed as 8/6 and 8/4.  otherwise :+1:

**Comment by lamagnifica** [comment]
@mhoeber I agree, but hoped that the change from "as of" to "week of" would make that less confusing. How should we word these and/or date these going forward, since multiple releases per week (and not just to platform) will become more and more the norm?

**Comment by mhoeber** [comment]
@lamagnifica   yes I see.  I think it's fine and a pattern we can keep

**Pull request merged**